lib/Db/SynchronizationContractMapper.php

```diff
@@ -234,7 +234,7 @@ public function updateFromArray(int $id, array $object): SynchronizationContract
      *
      * @return SynchronizationContract|null The matching contract or null if not found.
      */
-    public function findByOriginId(string $originId): ?SynchronizationContract
+    public function findByOriginId(?string $originId): ?SynchronizationContract
```

Comment:
If $originId optional doesnt this need a default value? Why would $originId be optional here?